deployments/chef/README.md

```diff
@@ -0,0 +1,158 @@
+# Splunk OpenTelemetry Collector Cookbook
+
+This cookbook installs and configures the Splunk OpenTelemetry Collector to
+collect metrics, traces and logs from Linux and Windows machines and sends
+data to [Splunk Observability Cloud](
+https://www.splunk.com/en_us/observability.html).
+
+## Prerequisites
+
+- [Splunk Access Token](https://docs.splunk.com/Observability/admin/authentication-tokens/org-tokens.html)
+- [Splunk Realm](https://dev.splunk.com/observability/docs/realms_in_endpoints/)
+- [Double-check exposed ports](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/security.md#exposed-endpoints) 
+  to make sure your environment doesn't have conflicts. Ports can be changed in the collector's configuration.
+
+## Linux
+
+Currently, the following Linux distributions and versions are supported:
+
+- Amazon Linux: 2
+- CentOS / Red Hat / Oracle: 7, 8
+- Debian: 9, 10, 11
+- SUSE: 12, 15 (**Note:** Only for Collector versions v0.34.0 or higher. Log collection with Fluentd not currently supported.)
+- Ubuntu: 18.04, 20.04, 22.04 (**Note:** Log collection with Fluentd [not currently supported on Ubuntu 22.04](https://www.fluentd.org/blog/td-agent-v4.3.1-has-been-released).)
+
+## Windows
+
+Currently, the following Windows versions are supported:
+
+- Windows Server 2019 64-bit
+- Windows Server 2022 64-bit
+
+## Usage
+
+To install the Collector and Fluentd, include the
+`splunk-otel-collector::default` recipe in the `run_list`, and set the
+attributes on the node's `run_state`. Below is an example to configure the
+required `splunk_access_token` attribute and some optional attributes:
+```yaml 
+{
+    "splunk-otel-collector": {
+        "splunk_access_token": "testing123",
+        "splunk_realm": "test",
+        "splunk_ingest_url": "https://ingest.test.signalfx.com",
+        "splunk_api_url": "https://api.test.signalfx.com",
+        "splunk_service_user": "splunk-otel-collector",
+        "splunk_service_group": "splunk-otel-collector",
+        "with_fluentd": true
```

Comment:
Maybe limit the initial example since most of these aren't necessary?

```suggestion
        "splunk_access_token": "<SPLUNK_ACCESS_TOKEN>",
        "splunk_realm": "<SPLUNK_REALM>",
```